src/scss/content.scss

```diff
@@ -0,0 +1,102 @@
+//////////////Carousel text animation////////////////////////////
+
+.animated {
```

Comment:
Pls, move styles related to animation to separate file, like animation.scss or mixin.scss. If to mixin.scss you shod update it to minix https://sass-lang.com/documentation/at-rules/mixin